src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,57 @@
+.stopwatch {
+  position: relative;
+  width: $watch-size;
+  height: $watch-size;
+  border: $border-size dotted $border-color;
+  border-radius: $circle;
+  display: flex;
+  justify-content: center;
```

Comment:
It is recommended to group related properties together. In this case, position and display properties are not directly related. You may want to consider grouping properties like width, height and border together, and position, display, justify-content and align-items together.